khal/khalendar/vdir.py

```diff
@@ -29,13 +31,13 @@ def __get__(self, obj, cls):
         return result
 
 
-def to_unicode(x, encoding='ascii'):
+def to_unicode(x: Union[str, bytes], encoding='ascii'):
     if not isinstance(x, str):
         return x.decode(encoding)
     return x
```

Comment:
Slightly off-topic: this function is a leftover from the python2 days. See https://github.com/pimutils/khal/pull/1155. Sorry for the conflict.